indico/modules/events/contributions/controllers/management.py

```diff
@@ -340,7 +353,18 @@ def _process(self):
 
 
 class RHEditSubContribution(RHManageSubContributionBase):
-    """Edit the subcontribution."""
+    """
+    Edit the subcontribution.
```

Comment:
```suggestion
    """Edit a subcontribution.
```

the "header' goes into the `"""` line